.golangci.yml

```diff
@@ -0,0 +1,106 @@
+run:
+  skip-files:
+    - pkg/dns/dns.go
+
+linters-settings:
+  dupl:
+    threshold: 100
+  funlen:
+    lines: 100
+    statements: 50
+  goconst:
+    min-len: 2
+    min-occurrences: 3
+  gocritic:
+    enabled-tags:
+      - diagnostic
+      - experimental
+      - opinionated
+      - performance
+      - style
+    disabled-checks:
+      - dupImport # https://github.com/go-critic/go-critic/issues/845
+      - ifElseChain
+      - octalLiteral
+      - whyNoLint
+      - wrapperFunc
+  gocyclo:
+    min-complexity: 15
+  gomnd:
+    # don't include the "operation" and "assign"
+    checks:
+      - argument
+      - case
+      - condition
+      - return
+    ignored-numbers:
+      - '0'
+      - '1'
+      - '2'
+      - '3'
+    ignored-functions:
+      - strings.SplitN
+
+  lll:
+    line-length: 140
+  misspell:
+    locale: US
```

Comment:
Aww, not en_CA? :p

Reply:
I don't believe that's a thing. The available options are UK or US. Canada is closer to US spelling than UK.

Reply:
Yeah, I don't think the support anything other than UK and US https://github.com/istio/istio/blob/master/.gitignore